test/fixtures/hello-cloud/template-runner.yml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
+kind: PodTemplate
+metadata:
+  name: hello-cloud-template-runner
+  labels:
+    name: hello-cloud-template-runner
+    app: hello-cloud
+template:
+  metadata:
+    name: task-runner
+    labels:
+      name: runner
+      app: hello-cloud
+  spec:
+    containers:
+    - name: task-runner
+      image: hello-world:latest
```

Comment:
How does the deploy upload a new `template-runner`? Where does that happen? I don't see the code for it